docker-compose.yml

```diff
@@ -25,6 +25,7 @@ services:
       - NAME=TechnoTim
       - BIO=Hey! Just a place where you can connect with me!
       - FOOTER=Thanks for stopping by!
+      - APPLEPODCASTS=https://podcasts.apple.com/de/podcast/geekparents/id000000000
```

Comment:
Is there a more generic link we can use here rather than `geekparents`?

Reply:
Oh I'm Sorry, this was my test-Data. I will fix it (with the typos)